bot/cogs/queue.py

```diff
@@ -44,7 +44,7 @@ async def update_last_msg(self, ctx, embed):
         self.last_queue_msgs[ctx.guild] = await ctx.send(embed=embed)
 
     @commands.command(brief='Join the queue')
-    @commands.max_concurrency(1, per=commands.BucketType.guild, wait=True)  # Only process one command per guild at once
+    #@commands.max_concurrency(1, per=commands.BucketType.guild, wait=True)  # Only process one command per guild at once
```

Comment:
You can delete this line altogether. I was hoping to remove it at some point in the future after I've handled the issues with race conditions, but I'll let those pop up and we can squash them as they appear.